content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,147 @@
+  connection established SHOULD be over a secure and authenticated channel.
+- Unused receivers and exporters SHOULD be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and Exporters MAY expose buffer, queue, payload, and/or worker
+  settings via configuration parameters. If these settings are available, users
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values may expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
+
+It is possible that a receiver MAY require the OpenTelemetry Collector run in a
+privileged mode in order to operate, which could be a security concern, but
+today this is not the case.
+
+### Safeguards against denial of service attacks
+
+Users should bind receivers' servers to addresses that limit connections to
+authorized users. <!-- Why? -->
+
+For example, if the OTLP receiver OTLP/gRPC server only has local clients, the
+`endpoint` setting SHOULD be bound to `localhost`:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+```
+
+Generally, `localhost`-like addresses should be preferred over the `0.0.0.0`
+address. For more information, refer to the
+[CWE-1327: Binding to an Unrestricted IP Address](https://cwe.mitre.org/data/definitions/1327.html)
+definition.
+
+### Encryption and Authentication
+
+<!-- TODO: SHOULD use encryption and authentication. How do you configure that?
+
+  **NOTE**: MAY pose a security risk if configuration parameters are modified improperly -->
+
+## Processors
+
+Processors sit between receivers and exporters. They are responsible for
+processing the data in some way. From a security perspective, they are useful in
+a few ways.
+
+### Recommended Processors
+
+<!--- TODO: SHOULD configure recommended processors. If so, what are they? -->
+
+### Scrubbing sensitive data
+
+It is common for an OpenTelemetry Collector to be used to scrub sensitive data
+before exporting it to a backend. This is especially important when sending the
+data to a third-party backend. The OpenTelemetry Collector **SHOULD** be
+configured to obfuscate or scrub sensitive data before exporting.
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+### Safeguarding around resource utilization
+
+Processors also offer safeguards around resource utilization. After implementing
+[safeguards around resource utilization in your hosting](/security/otel-collector-hosting-best-practices/),
+ensure your OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in otel-collector-hosting-best-practices -->
+
+The `batch` and especially `memory_limiter` processor help ensure that the
```

Comment:
We probably don't want to mention the memory limiter anymore, as we are in talks to deprecate/remove it.

cc @dmitryax

Reply:
Will remove the `memory_limiter` mention then. Thank you!